core/templates/dev/head/pages/exploration_player/AudioTranslationLanguageServiceSpec.js

```diff
@@ -71,4 +71,16 @@ describe('Audio translation language service', function() {
       explorationLanguageCode);
     expect(atls.getCurrentAudioLanguageCode()).toEqual('en');
   });
+
+  it('should not have any audio language option when no preferred language ' +
```

Comment:
Shouldn't it also have no audio language option even if a preferred language is available? If that's the case, probably should add a test for that too.

Reply:
Done!
(I realized that there is no use of  "when no preferred language is set" in the comment so removed.)